.github/maintainers_guide.md

```diff
@@ -15,7 +15,7 @@ $ brew update
 $ brew install pyenv
 ```
 
-Install necessary Python runtimes for development/testing. You can rely on Travis CI builds for testing with various major versions. https://github.com/slackapi/bolt-python/blob/main/.travis.yml
+Install necessary Python runtimes for development/testing. You can rely on Travis CI builds for testing with various major versions. <https://github.com/slackapi/bolt-python/blob/main/.travis.yml>
```

Comment:
```suggestion
Install necessary Python runtimes for development/testing. You can rely on GitHub Actions workflows for testing with various major versions. <https://github.com/slackapi/bolt-python/tree/main/.github/workflows>
```

One more change while this line is being reviewed 😉

Reply:
Nice 💯 good catch I just blindly followed the linters advice

Reply:
Super curious what linter this is? I love all of the changes 😄

Reply:
I believe this is [Markdown All In One](https://github.com/yzhang-gh/vscode-markdown) but I've testing out a few formatters and linters now have really wowed me 😞 

The Deno builtin one is good but requires that all our project use Deno so I don't think its an option

Would like to one day settle on one for all our projects